src/components/pages/order/Main/MainRightSide/Admin/OptionPanel.jsx

```diff
@@ -5,6 +5,8 @@ import styled from "styled-components";
 import { theme } from "../../../../../../theme";
 import { FiChevronUp, FiChevronDown } from "react-icons/fi";
 import { tabsConfig } from "./tabsConfig";
+import { AiOutlinePlus } from "react-icons/ai";
+import { MdModeEditOutline } from "react-icons/md";
 
 export default function OptionPanel() {
```

Comment:
```suggestion
export default function OptionsPanel() {
```
Je reverrai le nom de ce composant (et du fichier aussi par conséquent).

OptionsPanel (au pluriel) puisque tu as un map() dedans.

Mais dans l'idéal, je serai resté sur Tabs ou AdminTabs (voir mon commentaire plus loin sur Tab au lieu de Option).